src/components/VTabs/mixins/tabs-watchers.js

```diff
@@ -23,8 +23,8 @@ export default {
     right: 'callSlider',
     value (val) {
       const tab = this.tabs.find(tab => tab.action === val) || this.tabs[val]
```

Comment:
Change this to simply set the **lazyValue**. When the model changes externally, there is no need to modify the current **inputValue**. Since it pulls its value from **lazyValue**, we can simply update it, also clean up the code a bit.

```
value (val) {
  this.lazyValue = val
}
```

Reply:
done